### DISCUSSION TITLE
It doesn't work with my sdrplay1A.

### DISCUSSION THREAD

**Discussion by poetajuan** [AlexandreRouma/SDRPlusPlus#273] [Q&A]
Good afternoon, I try to make the sdr ++ work with the sdrplay1A and the sound comes out muted and without being able to control the radio. They can help me, I already installed the sdrplay api in the root directory.
Greetings and thanks.

**Comment by bickbick** [comment]
It defaults to very low gain...adjust LNA gain to about 90% and IF gain about 50%
